### ISSUE TITLE
Juice inline loader => ENOENT

### ISSUE THREAD

**Issue by davisford** [SamyPesse/tpl-emails#2]
I understand this has something to do with Juice, and it looks like they farm out the actual file loading to some other 3rd party lib, but with the following line in my template file:

```
  {% block head %}
  <link href="styles.css" media="all" rel="stylesheet" type="text/css" />
  {% endblock %}
```

Directory structure:

```
$tree src/server/email/
src/server/email/
├── index.js
└── templates
    └── my-template-name.html
```

and spinning up TplEmails like so in `src/server/email/index.js`

``` javascript
var nunjucks = require('nunjucks'),
  TplEmails = require('tpl-emails'),
  env = new nunjucks.Environment(new nunjucks.FileSystemLoader('./templates'));

return new TplEmails({
  render: env.render.bind(env)
});
```

I am left with this:

```
Template render error: Error: ENOENT, no such file or directory '/Users/davis/git/warehouse-web/file:/Users/davis/git/warehouse-web/src/server/email/templates/styles.css'
    at Error (native)
    at Object.fs.openSync (fs.js:500:18)
    at Object.fs.readFileSync (fs.js:352:15)
    at Object.util.getInlineFileContents (/Users/davis/git/warehouse-web/node_modules/tpl-emails/node_modules/juice/node_modules/web-resource-inliner/src/util.js:98:17)
    at Object.util.getTextReplacement (/Users/davis/git/warehouse-web/node_modules/tpl-emails/node_modules/juice/node_modules/web-resource-inliner/src/util.js:109:30)
    at Object.replaceLink (/Users/davis/git/warehouse-web/node_modules/tpl-emails/node_modules/juice/node_modules/web-resource-inliner/src/html.js:41:16)
```

Where it is pasting the root path in front of an absolute path.  Any idea -- or am I somehow initializing it incorrectly?

**Comment by davisford** [comment]
Looks like there is a [relativeTo](https://github.com/Automattic/juice/issues/104#issuecomment-74871476) option that can be passed which may resolve this

**Comment by SamyPesse** [comment]
If you fix it, can please do a PR :)

**Comment by davisford** [comment]
Sure, I am trying to resolve it right now.  I will send a PR

**Comment by SamyPesse** [comment]
Fixed by https://github.com/SamyPesse/tpl-emails/commit/b55b11097d0f08c41995ee706d98fe6d643bfae2 and released in **2.1.1**.
